core/src/main/java/org/pentaho/di/connections/ConnectionManager.java

```diff
@@ -295,27 +287,6 @@ public <T extends ConnectionDetails> boolean test( T connectionDetails ) throws
     return connectionProvider.test( connectionDetails );
   }
 
-  /**
```

Comment:
This method is specific to VFS connections.
Moved this to the new `VFSConnectionManagerWrapper` class.